src/plugins/unified_search/public/query_string_input/query_bar_menu.tsx

```diff
@@ -172,7 +186,12 @@ export function QueryBarMenu({
                 filters={filters!}
                 timeRangeForSuggestionsOverride={timeRangeForSuggestionsOverride}
                 onFiltersUpdated={onFiltersUpdated}
-                closePopover={closePopover}
+                onLocalFilterUpdate={setUpdatedFilter}
+                onLocalFilterCreate={setOriginalFilter}
+                closePopoverOnAdd={plainClosePopover}
+                closePopoverOnCancel={() => {
+                  closePopover();
```

Comment:
nit: is it was done on purpose? I see `closePopover` was declared as `useCallback` but here for some reason you anyway wrap it into new function

Reply:
Done.